### PR TITLE
Add latest dependency checker and notebook standizer

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

**Comment by thehomebrewnerd** [comment]
Do we want to run the notebook standardizer on this? I think typically we have left the outputs present in our demo notebooks so that people can visit the repo and read through the notebooks and see how they work without having to clone the repo, download the data and run (which can take significant time in some cases).

**Comment by gsheni** [comment]
@thehomebrewnerd Good point. Removed
https://github.com/alteryx/open_source_demos/pull/30/commits/c8f2368e318c273b696d92f0472fdeae278eed55

File: utils/notebook_version_standardizer.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import click
+import json
+import os
+
+DOCS_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), "source")
```
Comment (thehomebrewnerd) [inline utils/notebook_version_standardizer.py:5]: There is no `source` directory in this repo, so we will need to update this path

**Review by thehomebrewnerd — COMMENTED** [review]
Are you thinking that we will eventually try to implement some CI checks to run the notebooks when the versions get updates? Right now there are no CI tests in this repo so we won't get feedback on whether dependency updates break any notebooks, but that would be really nice to have!

**Comment by gsheni** [comment]
> Are you thinking that we will eventually try to implement some CI checks to run the notebooks when the versions get updates? Right now there are no CI tests in this repo so we won't get feedback on whether dependency updates break any notebooks, but that would be really nice to have!

yes, we do want to have some CI checks to run the notebooks when the Alteryx libraries get updated.

File: utils/notebook_version_standardizer.py (hunk 1 above)
Comment (gsheni) [reply utils/notebook_version_standardizer.py]: Decided to remove the notebook standizer

File: utils/latest_dependencies.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+evalml==0.39.0
+featuretools==1.3.0
+-e git+ssh://git@github.com/alteryx/woodwork.git@90beed66fcbcf9d790d3d5cfd9ad9660b8fde14e#egg=woodwork
```
Comment (thehomebrewnerd) [inline utils/latest_dependencies.txt:3]: This should be `woodwork==0.10.0`, no?

**Review by thehomebrewnerd — APPROVED** [review]

**Pull request merged**